internal/core/command/init.go

```diff
@@ -59,6 +59,10 @@ func Retry(useConsul bool, useProfile string, timeout int, wait *sync.WaitGroup,
 				// Setup Logging
 				logTarget := setLoggingTarget()
 				LoggingClient = logger.NewClient(internal.CoreCommandServiceKey, Configuration.Logging.EnableRemote, logTarget, Configuration.Logging.Level)
+				if err != nil {
+					ch <- err
+				}
```

Comment:
What's the purpose of those three lines? `err` always is `nil`...

Reply:
During an iteration of this newClient() returned an err and I removed the return but forgot to remove the error handling. Sorry about that!